dist/index.js

```diff
@@ -175,6 +176,33 @@ try {
     console.log("Keys added:");
     child_process.execSync('ssh-add -l', { stdio: 'inherit' });
 
+    child_process.execFileSync('ssh-add', ['-L']).toString().split(/\r?\n/).forEach(function(key) {
+        let parts = key.match(/\bgithub.com[:/](.*)(?:\.git)?\b/);
+
+        if (parts == null) {
+            return;
+        }
+
+        let ownerAndRepo = parts[1];
+        let sha256 = crypto.createHash('sha256').update(key).digest('hex');
+
```

Comment:
Nit: This obfuscates things when debugging/troubleshooting, which is why I had gone with the pseudo-host format.